bugwarrior/config/schema.py

```diff
@@ -321,41 +330,41 @@ def validate_config(config: dict, main_section: str, config_path: str) -> dict:
         # Convert top-level model to dict since target names are dynamic and
         # a bunch of calls to getattr(config, target) inhibits readability.
         return dict(bugwarrior_config_model(**config))
-    except pydantic.v1.ValidationError as e:
+    except pydantic.ValidationError as e:
         errors = ValidationErrorEnhancedMessages(e)
         raise_validation_error(str(errors), config_path, no_errors=len(errors))
 
 
 # Dynamically add template fields to model.
-_ServiceConfig = pydantic.v1.create_model(
-    '_ServiceConfig',
+_ServiceConfig = pydantic.create_model(
+    "_ServiceConfig",
+    __base__=BaseConfig,
     **{
-        f'{key}_template': (typing.Optional[str], None)
+        f"{key}_template": (typing.Optional[str], None)
         for key in taskw.task.Task.FIELDS
     },
 )
 
 
-class ServiceConfig(_ServiceConfig):  # type: ignore  # (dynamic base class)
-    """Pydantic_ base class for service configurations.
-
-    .. _Pydantic: https://docs.pydantic.dev/latest/
-    """
+# NOTE: I removed the deprecated fields
```

Comment:
These need to be added back. Like other deprecation warnings, they may have been in the code base for a long time but until a couple weeks ago there hadn't been a release for 5 years.